apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -56,7 +56,7 @@ const CurriculumCatalogCard = ({
     isTranslated={isTranslated}
     translationIconTitle={i18n.courseInYourLanguage()}
     isEnglish={isEnglish}
-    pathToCourse={pathToCourse}
+    pathToCourse={pathToCourse + '?viewAs=Instructor'}
```

Comment:
What should happen when a user is signed in as a student? (CC @dmcavoy )

Reply:
I double checked with Dani and she said it's fine in all cases to have that appended.

Reply:
Great, thanks!